wfe2/verify.go

```diff
@@ -337,7 +339,22 @@ func (wfe *WebFrontEndImpl) parseJWSRequest(request *http.Request) (*jose.JSONWe
 		return nil, probs.ServerInternal("unable to read request body")
 	}
 
-	return wfe.parseJWS(bodyBytes)
+	jws, prob := wfe.parseJWS(bodyBytes)
+	if prob != nil {
+		return nil, prob
+	}
+
+	// if the protected header contains a JWK and it's a EC key verify that it has
+	// proper field lengths in a goroutine as the caller doesn't care about the
```

Comment:
I'm surprised this is on a goroutine since it doesn't seem like it blocks on anything or does significant computation. What was the reason for doing it that way?

Reply:
There are no expensive computations, but since it has absolutely no effect on the caller I saw no particular reason to block on it doing anything ¯\_(ツ)_/¯.